CMakeLists.txt

```diff
@@ -133,14 +133,15 @@ SET (ORION_LIBS
     rest
     jsonParse
     jsonParseV2
-    apiTypesV2
+    rest
```

Comment:
However, repetition is a bit uncommon in our CMakeList.txt (maybe this is the first time we do it?). 

Thus, I'd suggest to add a short comment line to one of the rest occurrences to explain why it is duplicated. Otherwise, we have the risk to forget this, wrongly removing one of them in a future refactor, thus causing problems.